OrbitService/UtilsTest.cpp

```diff
@@ -152,4 +155,66 @@ TEST(Utils, ReadFileToString) {
   }
 }
 
+TEST(LinuxUtils, CategoriesTracepoints) {
+  using orbit_grpc_protos::TracepointInfo;
+  if (getuid() != 0) {
+    GTEST_SKIP() << "test is supported with root uid";
```

Comment:
That's nice, I wasn't aware of GTEST_SKIP.

Reply:
Actually, I found out that because I added GTEST_SKIP I no longer need to deal with the filesystem's exceptions since the code is not executed if it is not run with sudo and therefore there are not going to be any exceptions.

Reply:
as discussed offline .... guard against exceptions in real use anyway...